app/src/main/share-service-context.js

```diff
@@ -12,7 +12,13 @@ export default class ShareServiceContext {
 
     this.filePath = () => convert(Object.assign({}, exportOptions, {
       defaultFileName: this.defaultFileName
-    }));
+    })).catch(err => {
+      if (err.canceled) {
```

Comment:
I think this should be consistent and check the error name like done elsewhere here. Or maybe we should add a `canceled` property to `p-cancelable` errors? Could be nice for convenience.

Reply:
I can make it check for the name, and yes, that'd be nice 👍